machine/lenovo/lenovo_ne10032/rootconf/sysroot-lib-onie/sysinfo-platform

```diff
@@ -0,0 +1,25 @@
+
+get_serial_num()
```

Comment:
Thank you Curt. yes, this is not required.. I have removed it and placed another push ..